docs/console/company-configuration/extensions.md

```diff
@@ -68,35 +71,112 @@ On this page, you can:
+This configuration is possible in the detail page of the extension that, for _Composer_ extensions only, will include two tabs over the _General_ tab:
+the _Outbound calls_ tab and the _Internal calls_ tab.
 
-The *Extension Visibility* card shows in which contexts the extension is active and visible.
+### Outbound calls
 
-![edit extension visibility](./img/extensions/edit-extension-visibility.png)
+This tab allows you to configure the extension to communicate with other external APIs, by allowing to define a base URL to be called.
+It is also possible to define how to authenticate to the URL by including endpoint and the needed information for authentication.
 
 :::info
-Just as during creation, Visibility is tied to the Destination Area set for the extension.
-For example, if the Destination Area is set to *Company Overview*, the only available visibility option will be *Whole Company*.
+While it is possible to call multiple endpoints with the same base URL, it is possible to configure one outbound call for each extension.
+
+If you need to call multiple endpoints with different urls, you can create multiple extensions.
+:::
+
+When accessing to the _Outbound calls_ tab, if it has not been configured yet, an empty page with the button _Setup outbound calls_ will be shown.
+Clicking this button will open a modal where you can configure the outbound call.
+
+![Create a new outbound call](./img/extensions/extension-outbound-calls-create.png)
+
+The _Destination URL_ field is required and it is the base URL to be called.
+If you need to authenticate to this URL, keep the _Authentication required_ switch activated, then fill the _Authentication endpoint_ field with the endpoint to be called for authentication.
+
+The _Authentication type_ field will allow to select the type of authentication to be used. Currently supported are:
+
+- _Client credentials_ with Client ID and Client Secret
+- _Basic_ with Username and Password
+
+Once one of the available options is selected, below there will be the fields to fill in order to complete the authentication configuration, before to finally save with the _Save Changes_ button.
+
+![Details of a configured outbound call](./img/extensions/extension-outbound-calls-detail.png)
+
+After the creation, the outbound call configured will be shown in the page.
+Details will include the name of the URL to call and the authentication type used with some useful information.
+On the right side of the page there are menus to either update or delete the base URL, and change the credentials of the authentication.
+
+In the configuration page there will also be an URL next to the _Base URL to contact your APIs_ label. This URL is the one to be used to access the APIs from your endpoint.
+<!-- TODO: Are security reasons? Should I be more specific? -->
+In fact, for security reasons, the URL to be called is not the destination URL itself, but a different one that is the one that will be used to access the APIs
```

Comment:
How can I rephrase this to better explain the security reasons behind the choice to use a Proxy instead of using the _Destination URL_ itself?